web/react/utils/channel_intro_mssages.jsx

```diff
@@ -214,3 +173,15 @@ export function createStandardIntroMessage(channel, showInviteModal) { //eslint-
         </div>
     );
 }
+
+function createSetHeaderButton(channel) { // eslint-disable-line react/no-multi-comp
```

Comment:
Oh yeah, I forgot about that shortcut. What are your thoughts on turning on the ignoreStateless option for that eslint rule? We use helper functions like this all over the place inside of components, but the rule is only throwing an error now since not a method of a class.